src/Npgsql/NpgsqlReadBuffer.cs

```diff
@@ -107,7 +102,7 @@ internal TimeSpan Timeout
             Connector = connector;
             Underlying = stream;
             _underlyingSocket = socket;
-            _currentTimeout = TimeSpan.Zero;
+            _timeoutCts = new TimeoutCancellationTokenSourceWrapper(System.Threading.Timeout.InfiniteTimeSpan);
```

Comment:
nit: can do `using InfiniteTimeSpan = System.Threading.Timeout.InfiniteTimeSpan` above.

Reply:
Thanks!
Anyhow, that'll no longer be relevant once I revert to previous infinity semantics.

Reply:
This actually doesn't work as you can only change type names this way and not property names, but `using static System.Threading.Timeout;` did the trick (and it only imports the `InfiniteTimeSpan` and the `Infinite` property, in case you're afraid of spamming the global namespace).